velox/connectors/hive/HiveConnectorUtil.cpp

```diff
@@ -921,6 +921,21 @@ std::optional<std::string> getTimestampTimeZone(
   return std::nullopt;
 }
 
+parquet::WriterOptions::DataPageVersion getParquetDataPageVersion(
+    const config::ConfigBase& config,
+    const char* configKey) {
+  if (const auto version = config.get<std::string>(configKey)) {
+    if (version == "PARQUET_1_0") {
+      return parquet::WriterOptions::DataPageVersion::V1;
+    } else if (version == "PARQUET_2_0") {
+      return parquet::WriterOptions::DataPageVersion::V2;
+    } else {
+      VELOX_FAIL("Unsupported parquet datapage version {}", version.value());
+    }
+  }
+  return parquet::WriterOptions::DataPageVersion::kDefault;
```

Comment:
The return value must be optional. We can then remove the `kDefault` enum

Reply:
Done.